suite-native/ethereum-tokens/src/ethereumTokensSelectors.ts

```diff
@@ -6,14 +6,14 @@ import { TokenInfo } from '@trezor/blockchain-link';
 import { EthereumTokenAccountWithBalance } from './types';
 import { isEthereumAccountSymbol } from './utils';
 
-const filterTokenHasBalance = (token: TokenInfo) => !!token.balance && token.balance !== '0';
+export const filterTokenHasBalance = (token: TokenInfo) => !!token.balance && token.balance !== '0';
 
 export const selectEthereumAccountsTokensWithBalance = (
```

Comment:
IMHO since it is selecting only one account, the name should be:
```suggestion
export const selectEthereumAccountTokensWithBalance = (
```

Reply:
It's not selecting account, but accounts tokens with balance. The s is normal english 😅

Same way as you would say selectPetrsComments - you are selecting his comments, not comments from multiple Petrs 😅

Reply:
Sure but if you would want to select All tokens of all Eth accounts you would name it just the same. Seems confusing to me.

Reply:
You would just name it `selectAllEthereumAccountsTokensWithBalance`. I wouldn't worry about it too much